source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+.. command-block::
+
+   qiime tools import \
+     --type EMPPairedEndSequences \
+     --input-path emp-paired-end-sequences \
+     --output-path emp-paired-end-sequences.qza
+
+.. tip::
+   Links are included to view and download precomputed QIIME 2 artifacts and visualizations created by commands in the documentation. For example, the command above created a pared ``emp-paired-end-sequences.qza`` file, and a corresponding precomputed file is linked above. You can view precomputed QIIME 2 artifacts and visualizations without needing to install additional software (e.g. QIIME 2).
+
+.. qiime1-users::
+   In QIIME 1, we generally suggested performing demultiplexing through QIIME (e.g., with ``split_libraries.py`` or ``split_libraries_fastq.py``) as this step also performed quality control of sequences. We now separate the demultiplexing and quality control steps, so you can begin QIIME 2 with either multiplexed sequences (as we're doing here) or demultiplexed sequences.
+
+.. _`fmt cdiff demux`:
+
+Demultiplexing sequences
+------------------------
+
+To demultiplex sequences we need to know which barcode sequence is associated with each sample. This information is contained in the `sample metadata`_ file. You can run the following commands to demultiplex the sequences (the ``demux emp-paired`` command refers to the fact that these sequences are barcoded according to the `Earth Microbiome Project`_ protocol, and are pared-end reads). The ``demux.qza`` QIIME 2 artifact will contain the demultiplexed sequences. Additionally, we are passing the parameter ```--p-rev-comp-mapping-barcodes```, reverse complements the barcode sequences in the sample metadata prior to demultiplexing.
+
+.. command-block::
+
+    qiime demux emp-paired \
+      --i-seqs emp-paired-end-sequences.qza \
+      --m-barcodes-file sample-metadata.tsv \
+      --m-barcodes-column BarcodeSequence \
+      --o-per-sample-sequences demux.qza \
+      --p-rev-comp-mapping-barcodes
+
+After demultiplexing, it's useful to generate a summary of the demultiplexing results. This allows you to determine how many sequences were obtained per sample, and also to get a summary of the distribution of sequence qualities at each position in your sequence data.
+
+.. command-block::
+
+    qiime demux summarize \
+      --i-data demux.qza \
+      --o-visualization demux.qzv
+
+.. note::
+   All QIIME 2 visualizers (i.e., commands that take a ``--o-visualization`` parameter) will generate a ``.qzv`` file. You can view these files with ``qiime tools view``. We provide the command to view this first visualization, but for the remainder of this tutorial we'll tell you to *view the resulting visualization* after running a visualizer, which means that you should run ``qiime tools view`` on the .qzv file that was generated.
```

Comment:
or view with https://view.qiime2.org/

or look at the pre-computed QZVs that this tutorial will create if someone is following online

Reply:
just saw that you mention both of these below, but maybe it is worth merging that information together with this note.